client/driver/docker_test.go

```diff
@@ -20,11 +21,35 @@ func testDockerDriverContext(task string) *DriverContext {
 	return NewDriverContext(task, cfg, cfg.Node, testLogger())
 }
 
-// dockerLocated looks to see whether docker is available on this system before
-// we try to run tests. We'll keep it simple and just check for the CLI.
-func dockerLocated() bool {
-	_, err := exec.Command("docker", "-v").CombinedOutput()
-	return err == nil
+// dockerIsConnected checks to see if a docker daemon is available (local or remote)
+func dockerIsConnected() bool {
```

Comment:
Maybe pass the testing variable in and then instead of returning true/false you can use `t.Skip` and even put the reason. Also `t.Logf`